### PR TITLE
Crypto suggestions list

### PR DESCRIPTION
**Pull request by CristiArde**
crypto can be selected from the suggestion list

### CODE REVIEW

File: src/main/java/nitezh/ministock/StockSuggestions.java [hunk 1]
```diff
@@ -46,43 +46,76 @@ of this software and associated documentation files (the "Software"), to deal
...
+                    if (stockMatcher.matches() && symbol.length() < 8) {
+                        Map<String, String> suggestion = new HashMap<>();
+                        suggestion.put("symbol", symbol);
+                        suggestion.put("name", jsonO.getString("name"));
+                        suggestions.add(suggestion);
+                    }
+                }
             }
+            return suggestions;
+        } catch (JSONException ignored) {
```
Comment (Nick9500) [inline src/main/java/nitezh/ministock/StockSuggestions.java:116]: Is nothing supposed to happen when the exception is caught?

File: src/test/java/nitezh/ministock/activities/StockSuggestionsTest.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        cryptoJson = new JSONArray(result);
+
+        JSONObject jsonObject = cryptoJson.optJSONObject(0);
+        System.out.println(jsonObject.optString("name"));
+        assertTrue(!jsonObject.optString("name").isEmpty());
+        assertEquals("Bitcoin", jsonObject.optString("name"));
+        System.out.println(jsonObject.optString("symbol"));
+        assertTrue(!jsonObject.optString("symbol").isEmpty());
+        assertEquals("BTC", jsonObject.optString("symbol"));
+
```
Comment (Nick9500) [inline src/test/java/nitezh/ministock/activities/StockSuggestionsTest.java]: Whitespace

File: src/test/java/nitezh/ministock/domain/stocks.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+DF
```
Comment (Nick9500) [inline src/test/java/nitezh/ministock/domain/stocks.csv:1]: What is this file? Should it be committed to git?

**Review by Nick9500 — CHANGES_REQUESTED** [review]

File: src/test/java/nitezh/ministock/domain/stocks.csv (hunk 3 above)
Comment (CristiArde) [reply src/test/java/nitezh/ministock/domain/stocks.csv:1]: its for import tests while using an emulator.

File: src/main/java/nitezh/ministock/StockSuggestions.java (hunk 1 above)
Comment (CristiArde) [reply src/main/java/nitezh/ministock/StockSuggestions.java:116]: its needed for JSONArray

**Review by Nick9500 — APPROVED** [review]

**Pull request merged**